tests/unit/test_contrib/test_pydantic/test_pydanticdt_dto_factory.py

```diff
@@ -28,10 +28,7 @@ class TestModel(BaseModel):
         e: int = Field(default_factory=int_factory)
 
     field_defs = list(PydanticDTO.generate_field_definitions(TestModel))
-    assert (
-        field_defs[0].unique_model_name
-        == "tests.unit.test_contrib.test_pydantic.test_pydanticdt_dto_factory.test_field_definition_generation.<locals>.TestModel"
-    )
+    assert field_defs[0].model_name == "TestModel"
```

Comment:
Not related to this specific line, but is there any special meaning to the the `dt` part in this filename?  Should it just be `test_pydantic_dto_factory.py`?